compose/install.md

```diff
@@ -117,6 +117,68 @@ on Microsoft Windows Server and want to install Docker Compose.
 
 ### Install Compose on Linux systems
 
+You can install Docker Compose in different ways, depending on your needs:
+
+- Most users
+  [set up Docker's repositories](#install-using-the-repository) and install
+  from them, for ease of installation and upgrade tasks. This is the
+  recommended approach.
+
+- Some users download the binary and
+  [install it manually](#install-binary-manually) and manage
+  upgrades completely manually.
+
+- In testing and development environments, some users choose to use automated
+  [convenience scripts](#install-using-the-convenience-script) to install Docker.
```

Comment:
Perhaps we should move this one to the top of the list (it's not "best practice" for production environments, but it's the most common method, and with Compose targeting developer experience, I think it's ok to move it to the top)